conf/supervisord-dev.conf

```diff
@@ -0,0 +1,28 @@
+[supervisord]
```

Comment:
Bouncer didn't have a `supervisord-dev.conf`. `tox.ini` just ran `gunicorn` directly which was possible with Bouncer because it only really has one process that needs to run in dev. I just copy-pasted the `supervisord-dev.conf` from test-pyramid-app